hack/build/functests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+set -e
+
+source hack/build/common.sh
+source hack/build/config.sh
+
+functest_docker_prefix=${manifest_docker_prefix-${docker_prefix}}
```

Comment:
where is `${docker_prefix}` set?

Reply:
Left over I didn't remove, its not used anywhere (down side of copy and pasting stuff)

Reply:
Ah, okay.  Before we merge, it would be good to clean out kubevirt references and any dead code that won't be useful to CDI.